pyiron_atomistics/sphinx/base.py

```diff
@@ -2147,7 +2147,7 @@ def get_last(arr):
         if "positions" not in self.generic.list_nodes():
             self.generic.positions = np.array([self._job.structure.positions])
         if "cells" not in self.generic.list_nodes():
-            self.generic.cells = np.array([self._job.structure.cell.tolist()])
+            self.generic.cells = np.tile(np.array([self._job.structure.cell.tolist()]),(len(self.generic.energy_tot),1,1))
```

Comment:
```suggestion
np.tile(self._job.structure.cell,(len(self.generic.energy_tot),1,1))
```
This shouldn't have an effect, right?

Reply:
prolly not. But the unittest still fails.